src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -69,27 +71,39 @@ def validate_location(cmd, location):
             break
 
 
-def get_chart_path(registry_path, kube_config, kube_context, helm_client_location):
+def get_chart_path(registry_path, kube_config, kube_context, helm_client_location, chart_path_name='AzureArcCharts'):
     # Pulling helm chart from registry
     os.environ['HELM_EXPERIMENTAL_OCI'] = '1'
-    pull_helm_chart(registry_path, kube_config, kube_context, helm_client_location)
+    if chart_path_name == consts.Pre_Onboarding_Helm_Charts_Folder_Name:
+        pull_helm_chart(registry_path, kube_config, kube_context, helm_client_location, chart_path_name)
+    else:
+        pull_helm_chart(registry_path, kube_config, kube_context, helm_client_location, 'azure-arc')
 
     # Exporting helm chart after cleanup
-    chart_export_path = os.path.join(os.path.expanduser('~'), '.azure', 'AzureArcCharts')
+    chart_export_path = os.path.join(os.path.expanduser('~'), '.azure', chart_path_name)
     try:
         if os.path.isdir(chart_export_path):
             shutil.rmtree(chart_export_path)
     except:
-        logger.warning("Unable to cleanup the azure-arc helm charts already present on the machine. In case of failure, please cleanup the directory '%s' and try again.", chart_export_path)
-    export_helm_chart(registry_path, chart_export_path, kube_config, kube_context, helm_client_location)
+        logger.warning("Unable to cleanup the {} already present on the machine. In case of failure, please cleanup the directory '{}' and try again.".format(chart_path_name, chart_export_path))
+
+    if chart_path_name == consts.Pre_Onboarding_Helm_Charts_Folder_Name:
+        export_helm_chart(registry_path, chart_export_path, kube_config, kube_context, helm_client_location, chart_path_name)
+    else:
+        export_helm_chart(registry_path, chart_export_path, kube_config, kube_context, helm_client_location, 'azure-arc')
 
     # Returning helm chart path
-    helm_chart_path = os.path.join(chart_export_path, 'azure-arc-k8sagents')
-    chart_path = os.getenv('HELMCHART') if os.getenv('HELMCHART') else helm_chart_path
+    if chart_path_name == consts.Pre_Onboarding_Helm_Charts_Folder_Name:
+        helm_chart_path = os.path.join(chart_export_path, 'cluster-diagnostic-checks')
```

Comment:
I thought we removed the hyphen here, didn't we?

Reply:
its like AzureArcCharts folder has azure-arc-k8s agents, similarly PreOnboardingChecksCharts has cluster-diagnostic-checks helm release in it

Reply:
Ok